src/System.Diagnostics.EventLog/tests/EventLogReaderTests/EventLogRecordTests.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics.Eventing.Reader;
+using System.Threading;
+using Xunit;
+
+namespace System.Diagnostics.Tests
+{
+    public class EventLogRecordTests
+    {
+        private static EventLogRecord _sampleRecord;
+
+        static EventLogRecordTests()
```

Comment:
no need of static constructor , you can define a normal ctor because all the tests will run in a separate process. hence will even call this ctor again and again